_posts/2024-05-23-get-involved-with-pyos.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: single
+title: "Get Involved with pyOpenSci"
+excerpt: "Fun fact: there are a multitude of ways for you to get involved with pyOpenSci! From social media to in-person events to joining our editoral team, this guide walks you through every opportunity available."
+author: "Jesse Mostipak"
+permalink: /blog/get-involved-with-pyos.html
+header:
+  overlay_image: images/blog/2024/may/23-05-2024-get-involved.png
+  overlay_filter: 0.4
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+---
+## <i class="fa-regular fa-heart"></i> The pyOpenSci mission
+Founded in 2018 by our Executive Director, [Leah Wasser](https://github.com/lwasser), pyOpenSci builds diverse community that supports free and open Python tools for processing scientific data. We also build technical skills needed to contribute to open source and that support open science. While a diverse, inclusive community is at our core, radiating out from it are the three petals–how we accomplish our community goals–of pyOpenSci. These are:
+
+1. [Open peer review of scientific software](https://www.pyopensci.org/about-peer-review/index.html)
+2. [Community partnerships](https://www.pyopensci.org/partners.html)
+3. [Training and open educational resources](https://www.pyopensci.org/blog/pyos-education-announcement.html)
+
+A great primer for all things pyOpenSci is our [pyOpenSci lightning talk template [currently WIP]](https://docs.google.com/presentation/d/1205l8y75SI9spdklKnxDZzwwjArHvpeIpwFGZOvVLjc/edit?usp=sharing).
+
+## <i class="fa-solid fa-bullhorn"></i> Promote pyOpenSci online and in-person
+pyOpenSci is active on both [LinkedIn](https://www.linkedin.com/company/pyopensci) and [Fosstodon](https://fosstodon.org/@pyOpenSci), and you’re welcome to tag us on social media, as well as to [reach out with questions](mailto:media@pyopensci.org)! We also have [a series of brand assets and guidelines](https://docs.google.com/presentation/d/1glwf3BEPxo5H6pOHp6d55r2sbw8x-8kQZ9GzoLXwEDc/edit?usp=drive_link) if you need a copy of our logo, information on our fonts, and/or access to various illustrations we use.
+
+Dependent on timing, cost, and location, we’d also be more than happy to send you a set of pyOpenSci stickers to share! Please reach out to [media@pyopensci.org](mailto:media@pyopensci.org) to inquire about your event.
+
+Please feel free to use any information from [our website](https://www.pyopensci.org/) and our [pyOpenSci lightning talk template [currently WIP]](https://docs.google.com/presentation/d/1205l8y75SI9spdklKnxDZzwwjArHvpeIpwFGZOvVLjc/edit?usp=sharing), to share about the work pyOpenSci is doing. And don’t forget to share your personal experiences with pyOpenSci as well!
```

Comment:
how about something like... are ou going to a python meeting and want to give a lightning talk about pyOpenSci?... we can help... you can use any of the graphics or slides from this slide deck... 

but i think we need to make changes before people can use that deck

Reply:
agreed - I'm going to pull the talk template, and add in that they can reach out on socials or email us.